client/public/scripts.js

```diff
@@ -1,8 +1,8 @@
 // dom elements to manipulate
```

Comment:
Do we still need this file?

Reply:
Nope and I suppose visualiser.js and visualiser.html can also be dropped.